test/browser/features/fixtures/plugin_vue/typescript_vue2/index.html

```diff
@@ -11,7 +11,16 @@
     </div>
     <script>
       var el = document.getElementById('bugsnag-test-should-run')
-      el.textContent = el.innerText = typeof Proxy !== 'undefined'
+      var isLetSupported = false
+      var hasSafari10Bug = false
+      try {
+        eval('let a = "foo"')
+        isLetSupported = true
+        // See https://bugs.webkit.org/show_bug.cgi?id=171041
+        eval('function(a) { for(let a of []) { console.log(a) } }')
```

Comment:
Bug in Safari that is being triggered by later versions of the Vue2 bundle, which now include const/let/class etc. I'm not sure if this is a mistake by Vue2 as it seemed it should continue to support older browsers